File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/VehicleProfileCreationRequest.java
package ru.vitaliy.petrov.server.forms.requests;

import lombok.Data;

@Data
public class VehicleProfileCreationRequest {
    private final String vehicleBrand;
    private final String vehicleVIN;
    private final String vehicleRegistrationSign;
    private final String vehicleRegistrationCertificate;
    private final String vehicleOwnerFullName;
    private final String vehicleOwnerResidentialAddress;
    private final String vehicleInsurancePolicyNumber;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/VehicleProfileCreationRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

data class VehicleProfileCreationRequest(
    val vehicleBrand: String,
    val vehicleVIN: String,
    val vehicleRegistrationSign: String,
    val vehicleRegistrationCertificate: String,
    val vehicleOwnerFullName: String,
    val vehicleOwnerResidentialAddress: String,
    val vehicleInsurancePolicyNumber: String
)
